spec/controllers/submission_controller_spec.rb

```diff
@@ -2,4 +2,22 @@
 
 RSpec.describe SubmissionController, type: :controller do
 
+  before :each do
+    request.env['CONTENT_TYPE'] = 'application/json'
+    allow_any_instance_of(ApplicationController).to receive(:verify_token!)
+  end
+
+  it 'creates a submission' do
+    post :create, body: {somthing: 1}.to_json
+
+    expect(Submission.all.count).to eq(1)
+  end
+
+  it 'starts a delayed job' do
+    post :create, body: {somthing: 1}.to_json
+
+    expect(response).to have_http_status(201)
+    expect(Delayed::Job.all.count).to eq(1)
+  end
+
```

Comment:
Fairly certain this behaviour is tested elsewhere given we are re-using the class